tools/generator/InterfaceGen.cs

```diff
@@ -375,9 +376,10 @@ void GenerateEventArgs (Method m, StreamWriter sw, string indent, CodeGeneration
 						if (p.IsSender)
 							continue;
 						sw.WriteLine ();
-						sw.WriteLine ("{0}\t{1} {2};", indent, opt.GetOutputName (p.Type), opt.GetSafeIdentifier (p.Name));
+						var safeTypeName = p.Type.StartsWith ("params ", StringComparison.Ordinal) ? p.Type.Substring (7) : p.Type;
```

Comment:
This usage makes the `7` a bit more obvious, as `"params "` is on the same line/nearby (unlike the previous spot), but I would still prefer use of `"params ".Length` over a hardcoded `7`.